server/controllers/login.js

```diff
@@ -0,0 +1,48 @@
+const bcrypt = require("bcrypt");
+const { sign } = require("jsonwebtoken");
+const getPassword = require("../database/queries/getPassword");
+require("dotenv").config();
+const { SECRET } = process.env;
+const createToken = (email, secret) => {
+  return sign({ email }, secret);
+};
+
+exports.login = (req, res) => {
+  const { email, password } = req.body;
+  if (!email || !password) {
+    return res
+      .status(200)
+      .json({ message: 'all fields are required', status: '400' });
+  }
+  getPassword(email, (err, result) => {
+    if (err) {
+      console.log(err);
+    } else {
+      if (result) {
+        const hash = result.password;
+        bcrypt
+          .compare(password, hash)
+          .then((result) => {
+            if (result) {
+              const token = createToken(email, SECRET);
+              res
+                .cookie('token', token, { maxAge: 900000, httpOnly: true })
+                .json({ status: 'sucess', token });
```

Comment:
status will be indicated by the HTTP status (200), no need for it here. Also, `token` will be handled automatically by the browser through the cookie, no point in including it in the response message. So it's enough to just do `.status(200)` instead of this..